packages/dd-trace/src/appsec/rule_manager.js

```diff
@@ -134,6 +153,11 @@ function updateWafFromRC ({ toUnapply, toApply, toModify }) {
     config.apply_state = newApplyState
     if (newApplyError) config.apply_error = newApplyError
   }
+
+  if (newActions.modified) {
+    blocking.updateBlockingConfiguration(concatArrays(newActions).filter(action => action.id === 'block'))
```

Comment:
```suggestion
    blocking.updateBlockingConfiguration(concatArrays(newActions).find(action => action.id === 'block'))
```

Reply:
I want to send an array, and let to the `updateBlockingConfiguration` the responsability to select the good blocking page.

Reply:
But... after think in that again... I'll change it